src/odin/maneuversbuilder.cc

```diff
@@ -352,6 +352,25 @@ void ManeuversBuilder::Combine(std::list<Maneuver>& maneuvers) {
         curr_man = next_man;
         ++next_man;
       }
+      // Combine ramp maneuvers
+      else if (AreRampManeuversCombinable(prev_man, curr_man, next_man)) {
+
+        uint32_t turn_degree = GetTurnDegree(prev_man->end_heading(), curr_man->begin_heading());
+        Turn::Type turn_type = Turn::GetType(turn_degree);
+        if (turn_type == Turn::Type::kStraight && curr_man->length() < 0.05f) {
```

Comment:
Can `0.05f` be a named constant? Also, how was 50m chosen?

Reply:
@danpaz of course.  this was just a test.

Reply:
@danpaz this PR should've been marked as a draft....we wanted to see the diffs that this created.

Reply:
Ok cool I'll hold off for now 😄